### DISCUSSION TITLE
Consider collapsing Scratch Dev tools?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#361] [Discussion]
Scratch Dev tools (by griffpatch) is an amazing extension, but it has so much functionality. Perhaps collapse it into one 'addon' in settings with the parts of it being able to be enabled with check boxes? Another thing, where should the official repo for Dev Tools be? Here or where it is in griffpatch's repo?

**Comment by Explosion-Scratch** [comment]
Whoops! I forgot to cc griffpatch! Here you go: cc @griffpatch  What do you think?

**Comment by WorldLanguages** [comment]
Well, I don't think it's worth it to try to understand the code for that so unless griffpatch splits the code it's likely going to stay like that

**Comment by Explosion-Scratch** [comment]
Okay! But should we add check boxes for the individual parts?

**Comment by WorldLanguages** [comment]
@Explosion-Scratch We'd have exactly the same problem

**Comment by Explosion-Scratch** [comment]
Oops lol, stupid me

**Comment by Sqrts321** [comment]
GRIFFPATCH
